main.js

```diff
@@ -561,27 +557,29 @@
 
     function addEventListeners() {
         document.querySelectorAll('button[data-method-name]').forEach((button) => {
-            button.addEventListener('click', (e) => {
-                let methodName = e.target.getAttribute('data-method-name');
-                let args = document.querySelectorAll(`.result__${methodName} [data-arg-name]`);
-                let argsValues = [];
-                let valid = true;
-                args.forEach((arg) => {
-                    arg.classList.remove('invalid');
-                    const argType = arg.getAttribute('data-arg-type');
-                    const validateRule = dataTypesRegExp[argType];
-                    if (validateRule && !validateRule.test(arg.value)) {
-                        valid = false;
-                        arg.classList.add('invalid');
+            button.addEventListener('click', (event) => {
+                const methodName = event.target.getAttribute('data-method-name');
```

Comment:
Might be better to extract eventHandler function. We should be able to remove event listener in future.

Reply:
Don't mind